sections/desktop-img.liquid

```diff
@@ -0,0 +1,26 @@
+<div class="desktop-images">
+    <img src="{{section.settings.desktop_image_1 | img_url: '800x'}}">
+    <img src="{{section.settings.desktop_image_2 | img_url: '800x'}}">
+    <img src="{{section.settings.desktop_image_3 | img_url: '800x'}}">
+</div>
+
+{% schema %}
+{
+  "name": "desktop-img",
+  "settings": [
```

Comment:
could you use blocks to build this, please?

Resource
[Blocks](https://shopify.dev/tutorials/develop-theme-use-sections#blocks)